src/components/pages/DataHub/DataHub.js

```diff
@@ -207,9 +207,7 @@ const DataHub = props => {
         added, more data can be visualized.
       </h3>
       <h2 className="h1Styles">Approvals vs Denials Nationwide</h2>
-      <p className="divider">
-        <Icon component={() => <img src={OrangeLine} alt="divider icon" />} />
-      </p>
+      <p className="divider"></p>
```

Comment:
What is the purpose of this p tag?

Reply:
Not sure, I didn't add that and I didn't notice it before. I will remove it on the next push

Reply:
If you look, the p tag was there before, you just removed the Orange Icon component and left it's parent component, that no longer serves a purpose